src/modules/xml/mlt-xml.dtd

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='utf-8'?>
 
-<!-- MLT XML DTD v0.2.0 -->
```

Comment:
I am thinking perhaps we should change the DTD version to a major and minor only that reflects the MLT version. We do not need to increment it with each minor version of MLT, but anytime we break the schema by changing it in a non-backwards compatible way, it should coincide with a major new version of MLT. And anytime we add something in a backwards compatible way or fix the schema bump the minor version. The XML interface is nearly as important as the ABI. See how in Shotcut I am now using the MLT major version reflected in the MLT XML to check compatibility.

Reply:
I also need to add chain and link types to the dtd before the official release.

Shall I change the DTD version to 7.0.0?

Reply:
yes, change it to 7.0.0

Reply:
Done